src/components/AddFood.js

```diff
@@ -0,0 +1,34 @@
+import React    from 'react'
+
+function AddFood(props){
+    return (
+        <form onSubmit={props.onAdd}>
+            <div className="field">
```

Comment:
this way of passing information to the function via event.currentTarget works good. But in react it is more common working around the state. So having states for the different input fields that change dynamically with an onChange method. Then you can invoke the function with the information from the state.